samples/sdfw/ssf_client/Kconfig.sysbuild

```diff
@@ -0,0 +1,16 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+source "${ZEPHYR_BASE}/share/sysbuild/Kconfig"
+
+config BUILD_ALSO_FOR_RADIO_CORE
+	bool "Build also for the radio CPU"
+
+config RADIO_IMAGE_BOARD
+	string "The board used for cpurad target"
+	depends on BUILD_ALSO_FOR_RADIO_CORE
+	default "nrf54h20dk/nrf54h20/cpurad" if BOARD_NRF54H20DK_NRF54H20_CPUAPP
+	default "nrf9280pdk/nrf9280/cpurad" if BOARD_NRF9280PDK_NRF9280_CPUAPP
```

Comment:
Rework to work as the sysbuild Kconfig files do i.e. rename to `*_BOARD_TARGET_CPUCLUSTER`, reference `SOC_*` Kconfigs and only have the CPU cluster here then work out the board to use properly in cmake - see https://github.com/nrfconnect/sdk-nrf/blob/main/sysbuild/Kconfig.netcore#L15 and https://github.com/nrfconnect/sdk-nrf/blob/main/sysbuild/netcore.cmake

Reply:
Thanks. I have updated to work out the radio board target in cmake.